projects/sunbird-video-player/src/lib/components/video-player/video-player.component.ts

```diff
@@ -13,8 +13,9 @@ import { ViewerService } from '../../services/viewer.service';
   styleUrls: ['./video-player.component.scss'],
   encapsulation: ViewEncapsulation.None
 })
-export class VideoPlayerComponent implements AfterViewInit, OnInit, OnDestroy {
+export class VideoPlayerComponent implements AfterViewInit, OnInit, OnDestroy, OnChanges {
   @Input() config: any;
+  @Input() action: string;
```

Comment:
Can we make this an interface with action name and other properties,
**Example:**
Tomorrow if we get a use case like play from 1:45 min if answered all questions in question set or play from 1:10 min if unanswered

Also, please update the documents both README and [microsite](https://knowlg.sunbird.org/learn/product-and-developer-guide/player/v2/video-player) and add the unit test cases for the same/

Reply:
We have changed it to a property in an Interface , updated readme but I dont have access to microsite